server/sonar-server/src/main/java/org/sonar/server/permission/ws/PermissionsWsParametersBuilder.java

```diff
@@ -57,14 +58,14 @@ private PermissionsWsParametersBuilder() {
     // static methods only
   }
 
-  public static void createPermissionParameter(NewAction action) {
-    action.createParam(PARAM_PERMISSION)
+  public static NewParam createPermissionParameter(NewAction action) {
+    return action.createParam(PARAM_PERMISSION)
       .setDescription(PERMISSION_PARAM_DESCRIPTION)
       .setRequired(true);
   }
 
-  public static void createProjectPermissionParameter(NewAction action) {
-    action.createParam(PARAM_PERMISSION)
+  public static NewParam createProjectPermissionParameter(NewAction action) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUndocumentedApi)